package.json

```diff
@@ -13,6 +13,7 @@
     "npm": "3.10.10"
   },
   "scripts": {
+    "seeder": "node ./node_modules/.bin/babel-node seeder/seedDatabase",
```

Comment:
Why are we using babel node for this?

Reply:
I noticed if i run the script without babel node it will be complaining of not understanding es6 syntax. i tried running it through the dist folder path without babel node and i found out that it is working but it means before the database can be seeded the application needs to be started once.